### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by WalDragnir**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,12 @@
-# write your code here
+from pathlib import Path
+from shutil import move
+
+
+def move_file(command: str) -> None:
+    input_command, source_path, move_path = command.split()
```
Comment (ArsenPidhoretskyi) [inline app/main.py:6]: you code will raise error without any info if command will be one of:
mv file1.txt
mv file1.txt file2.txt file3.txt

File: app/main.py [hunk 2]
```diff
@@ -1 +1,12 @@
...
+    if input_command != "mv":
+        raise ValueError("The command should be 'mv'")
+    elif "/" in move_path:
+        path = move_path[:move_path.rfind("/")]
+        Path(path).mkdir(parents=True, exist_ok=True)
```
Comment (ArsenPidhoretskyi) [inline app/main.py:11]: Use os.path.split for check

File: app/main.py [hunk 3]
```diff
@@ -1 +1,12 @@
...
+    move(source_path, move_path)
```
Comment (ArsenPidhoretskyi) [inline app/main.py:12]: No need to use shututil create your custom solution

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 4]
```diff
@@ -1 +1,13 @@
...
+
+def move_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("The command must be 'mv file.txt dir/new.txt'")
+    input_command, source_path, move_path = command.split()
+    if input_command != "mv":
+        raise ValueError("The command should be 'mv'")
+    head_tail = os.path.split(move_path)
+    if head_tail[0] != "" and not os.path.exists(head_tail[0]):
+        os.makedirs(head_tail[0])
```
Comment (Grayder0152) [inline app/main.py:12]: use argument `exists_ok` in `makedirs`

**Review by Grayder0152 — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — APPROVED** [review]

**Pull request closed without merging**